modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/PaginationBarTag.java

```diff
@@ -186,25 +196,32 @@ protected int processStartTag() throws Exception {
 
 		JspWriter jspWriter = pageContext.getOut();
 
-		jspWriter.write("<div class=\"dropdown pagination-items-per-page\">");
-		jspWriter.write("<button class=\"dropdown-toggle btn btn-unstyled\"");
-		jspWriter.write(" type=\"button\">");
-		jspWriter.write(_activeDelta.toString());
-		jspWriter.write(StringPool.SPACE);
-
 		ResourceBundle resourceBundle = TagResourceBundleUtil.getResourceBundle(
 			pageContext);
 
-		jspWriter.write(
-			StringUtil.toLowerCase(LanguageUtil.get(resourceBundle, "items")));
+		if (_showDropDown) {
+			jspWriter.write(
+				"<div class=\"dropdown pagination-items-per-page\">");
+			jspWriter.write(
+				"<button class=\"dropdown-toggle btn btn-unstyled\"");
+			jspWriter.write(" type=\"button\">");
+			jspWriter.write(_activeDelta.toString());
+			jspWriter.write(StringPool.SPACE);
+
+			jspWriter.write(
+				StringUtil.toLowerCase(
+					LanguageUtil.get(resourceBundle, "items")));
 
-		IconTag iconTag = new IconTag();
+			IconTag iconTag = new IconTag();
```

Comment:
Nice :)

It solved an old doubt that I used to have about using tags within another